tests/Doctrine/Tests/Models/Upsertable/JoinedInheritanceNonUpdatableColumn.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Tests\Models\Upsertable;
+
+use Doctrine\ORM\Mapping\Column;
+use Doctrine\ORM\Mapping\Entity;
+use Doctrine\ORM\Mapping\Table;
+
+/**
+ * @Entity
+ * @Table(name="joined_inheritance_non_updatable_column")
+ */
+#[Entity]
+#[Table(name: 'joined_inheritance_non_updatable_column')]
+class JoinedInheritanceNonUpdatableColumn extends JoinedInheritanceRoot
```

Comment:
Can you please read and apply https://github.com/doctrine/orm/pull/10683#issuecomment-1537466852 ?

Reply:
Done.